README.md

```diff
@@ -4,40 +4,116 @@
+    - Resume from the latest checkpoint if the training was interrupted.
+  - Learning rate scheduling
+  - Backoff: Restore the best checkpoint and change the learning rate if the loss is not decreasing.
+  - [Hooks](padertorch/train/hooks.py): Extend the basic features of the trainer with your own functionality.
+  - **Test run**: The trainer has a `test_run` function to train the model for few iterations and test if
+      - the model is executable (burn test),
+      - the validation is deterministic/reproducible, and
+      - the model changes the parameter during training.
+  - **Virtual minibatch**:
+    - The `Trainer` usually does not know if the model is trained with a single example or multiple examples (minibatch), because the examples that are yielded from the dataset are directly forwarded to the model.
+    - When the `virtual_minibatch_size` option is larger than one, the trainer calls the forward and backward step `virtual_minibatch_size` times before applying the gradients. This increases the number of examples over which the gradient is calculated while the memory consumption stays similar, e.g., with `virtual_minibatch_size=2` the gradient is accumulated over two (batched) examples before calling `optimizer.step(); optimizer.zero_grad()`. See [here](doc/virtual_batch_size_multi_gpu.md) for a more thorough explanation.
+- **Logging**: As logging backend, we use [tensorboardX](https://github.com/lanpa/tensorboardX) to generate a `tfevents` file that can be visualized from a [tensorboard](https://github.com/tensorflow/tensorboard). Custom values to be logged can be defined in subclasses of `padertorch.Model`.
+- **Multi-GPU training**: Easily deploy your model onto multiple GPUs to increase the total batch size and speed up the training. See [here](doc/virtual_batch_size_multi_gpu.md#L68) for implementation details and the [example](padertorch/contrib/examples/multi_gpu) for how to enable it.
+- **Support for lazy data preparation**: Ever tired of pre-computing features, taking up time and space on the hard disk? Padertorch works with lazy dataloaders (e.g., [lazy_dataset](https://github.com/fgnt/lazy_dataset)) which extract the features on the fly!
+- **Hands-on examples**: We provide models and training scripts which help you getting started with padertorch.
+
+# Installation
+
+**Requirements**
+- Python 3
+- matplotlib
+- To run the examples:
+  - [sacred](https://github.com/IDSIA/sacred)
+  - torchvision
+  - [pb_bss](http://github.com/fgnt/pb_bss)
+- Installed by padetorch:
+  - torch
+  - tensorboardX
+  - einops
+  - tqdm
+  - natsort
+  - [lazy_dataset](https://github.com/fgnt/lazy_dataset)
+  - IPython
+  - [paderbox](https://github.com/fgnt/paderbox)
+
+```bash
+$ git clone https://github.com/fgnt/padertorch.git
+$ cd padertorch && pip install -e .
```

Comment:
Can you move this above `Requirements` and change `pip install -e .` to `pip install -e .[all]`?
Below that you may write, `This will install all dependencies, for a light installation you can drop "[all]"`